multinode-demo/start_nodes.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+nodes_file=$1
```

Comment:
Would to be good to check for missing arguments and display an error message / usage info